website/demo-styles/container.scss

```diff
@@ -28,9 +28,11 @@
     line-height: 160px;
   }
 
-  & > .source > .el-container {
+  & > .source  > div > .el-container {
     margin-bottom: 40px;
-
+    &:nth-child(7) {
+      margin-bottom: 0;
+    }
```

Comment:
Could you tell me about why is the magical number 7 this particular?

Reply:
The last `.el-container` element needs to set margin to 0, otherwise it does not display the same as other sample views.

Reply:
Then you need yo apply this to the last-child instead of 7th child

Reply:
That's right,  It's a better choice, It's changed.